tools/pod-utils/clonerefs.yaml

```diff
@@ -12,6 +12,7 @@ items:
         name: clonerefs:latest
     runPolicy: Serial
     source:
+      contextDir: prow/cmd/clonerefs
       git:
         ref: master
         uri: https://github.com/kubernetes/test-infra.git
```

Comment:
We are not using openshift/test-infra, mind updating both builds to use it?

Reply:
Or it can be done in a follow-up since I think none of the pod utilities are based off our fork.